patroni/api.py

```diff
@@ -759,6 +759,7 @@ class RestApiServer(ThreadingMixIn, HTTPServer, Thread):
     def __init__(self, patroni, config):
         self.patroni = patroni
         self.__listen = None
+        self.request_queue_size = config.get('request_queue_size', 5)
```

Comment:
```suggestion
        self.request_queue_size = int(config.get('request_queue_size', 5))
```
Just in case if the value in YAML is quoted.